pkgs/applications/graphics/imv/default.nix

```diff
@@ -41,7 +41,7 @@ let
   };
 
   backends = {
-    inherit freeimage libtiff libpng librsvg libheif;
+    inherit libtiff libpng librsvg libheif libjxl;
```

Comment:
I think you should keep `freeimage` in `backends` because it looks like `backends` lists the available backends in imv, and `withBackends` lists the ones we want to enable
```suggestion
    inherit freeimage libtiff libpng librsvg libheif libjxl;
```
`backends` is then used in the loop below and each backend is either enabled (if it is found in `withBackends`) or disabled (if it is not)

Reply:
Thanks, now I understand. I added `freeimage = false`, as `freeimage` can't stay in the dependency list, as it's marked  knowVulnerabilities. Maybe there is a better way.

Reply:
What's the concern leaving it in the dependency list? If it's not enabled, as far as I see, there's no issues.

Reply:
You are right.
I thought I had tested that, and I got `error: Package ‘freeimage-unstable-2021-11-01’ in […] is marked as insecure, refusing to evaluate.`, but I must have made a mistake in my test.
Let's fix it :)